scripts/fast-reboot

```diff
@@ -410,6 +410,12 @@ docker exec -i bgp pkill -9 zebra
 docker exec -i bgp pkill -9 bgpd || [ $? == 1 ]
 debug "Stopped  bgp ..."
 
+# Kill nat docker after saving the conntrack table
+debug "Stopping nat ..."
+/usr/bin/dump_nat_entries.py
+docker kill nat > /dev/null
```

Comment:
To avoid systemd restart the service automatically after kill you should add 'systemctl stop nat'

Reply:
Done.